### DISCUSSION TITLE
Filter rows with emty array column

### DISCUSSION THREAD

**Discussion by Tihomir971** [supabase/supabase#6403] [Questions] [answered]
I have int array column.
How to find rows with empty array column?

**Comment by GaryAustin1** [comment, marked as answer]
If you are using Javascript client:
.containedBy('tags',[])
or
.eq('tags','{}')

For SQL https://stackoverflow.com/questions/737669/postgres-how-to-check-for-an-empty-array

Note: this is for columns with [ ], null columns will not be returned.
